src/registrar/utility/csv_export.py

```diff
@@ -94,6 +97,11 @@ def parse_row(columns, domain_info: DomainInformation, security_emails_dict=None
     }
 
     if get_domain_managers:
+        for i in range(1, max_dm_count + 1):
```

Comment:
**non blockingsuggestion**
Hey so won't we ONLY need to add to this list if this check in write_csv actually enters and updates the max dm count values " if dm_count > max_dm_count:
                max_dm_count = dm_count"

Instead of always doing this call I would think just passing an update_domain_count bool could would save on looping through the columns. line 102 has O(n) complexity. Not ideal, but also this shouldn't hurt a ton given that list shouldn't be too big.